examples/test-dockerfile/dockerignore-default/Dockerfile

```diff
@@ -1,3 +1,7 @@
-FROM scratch
+FROM bash
 
 COPY . /
+
+RUN [ -f "app.txt" ]
+
+RUN [ ! -f "ignore.txt" ]
```

Comment:
Added these here and elsewhere to ensure we always build the image with the expected files.